src/item/GoldenAppleEnchanted.php

```diff
@@ -30,7 +30,7 @@ class GoldenAppleEnchanted extends GoldenApple{
 
 	public function getAdditionalEffects() : array{
 		return [
-			new EffectInstance(VanillaEffects::REGENERATION(), 600, 4),
+			new EffectInstance(VanillaEffects::REGENERATION(), 600, 2),
```

Comment:
the amplifier value should be 1, according to this
https://github.com/pmmp/PocketMine-MP/blob/stable/src/entity/effect/EffectInstance.php#L100

changelogs: https://feedback.minecraft.net/hc/en-us/articles/29937458432397-Minecraft-Beta-Preview-1-21-40-20

Reply:
I see it already applied 1 level so when a user inputs 0 it’ll automatically be 1.